filebeat/prospector/prospector_log.go

```diff
@@ -54,6 +56,8 @@ func (p *ProspectorLog) Init() {
 }
 
 func (p *ProspectorLog) Run() {
+	p.mutex.Lock()
```

Comment:
You are right, sorry I missed that. The mutex is not required - not sure why I missed that.